kuma/wiki/models.py

```diff
@@ -413,7 +413,9 @@ def get_zone_subnav_html(self):
         """
         src = self.get_zone_subnav_local_html()
         if (not src) and self.nearest_zone:
-            src = self.nearest_zone.document.get_zone_subnav_local_html()
+            document_id = self.nearest_zone.document_id
+            document = Document.admin_objects.only('html', 'rendered_html', 'zone_subnav_local_html').get(id=document_id)
```

Comment:
Why ``admin_objects`` (which returns deleted documents as well) instead of ``objects``?

Reply:
For some [historical purpose](https://bugzilla.mozilla.org/show_bug.cgi?id=1365960), we have seen some error in `DocumentNearestZoneJob` about already deleted document makes `500` error
Therefore used `admin_objects` instead.